ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -208,6 +210,17 @@ def start_learning(self, env_manager: EnvManager) -> None:
                             env_manager.external_brains[name]
                         )
                         self.start_trainer(trainer, env_manager)
+                        agent_manager = AgentManager(
+                            processor=AgentProcessor(
+                                trainer,
+                                trainer.policy,
+                                trainer.parameters["time_horizon"]
+                                if "time_horizon" in trainer.parameters
```

Comment:
replace with `trainer.parameters.get("time_horizon")`

Reply:
good call - I now get with a default value of sys.maxsize.